### ISSUE TITLE
Tab Panels on concept pages always expect a corresponding tab

### ISSUE THREAD

**Issue by paul-butcher** [wellcomecollection/wellcomecollection.org#10598]
**What**

On a [Concept page with tabs](https://wellcomecollection.org/concepts/d46ea7yk), the panels are correctly labelled by their corresponding tabs.  e.g. if you use a screen reader and enter the panel, it will read the tab's text as the section's title.

However, on a [Concept page that does not have tabs](https://wellcomecollection.org/concepts/fmydsuw2), the "tab panel" is still marked as being "labelled by" a non-existent tab. The reader will use its default announcement for entering a tab panel (in Chromevox, this says "section tabpanel")

Concept pages only have tabs when there are tabs to choose from, so a Genre like Lithographs, which only has content that uses it as a genre, and no content that uses it as a subject, is not expected to have tabs

Update: [This concept](https://wellcomecollection.org/concepts/qjg7htza) does not have tabs in images

**Comment by rcantin-w** [comment]
It's also removing the labelling of the section, so we don't know if the works/images are ABOUT or BY the subject.

**Comment by rcantin-w** [comment]
Timeboxed investigation (couple hours)
